apps/dashboard/src/composables/useUserType.js

```diff
@@ -25,16 +38,22 @@ export default function useUserType(userClaims) {
       return AUTH_USER_TYPE.SUPER_ADMIN;
     }
 
-    if (claims?.role === UserRoles.LAUNCH_ADMIN) {
-      return AUTH_USER_TYPE.LAUNCH_ADMIN;
-    }
+    if (claims?.role) {
+      if (claims?.role === UserRoles.LAUNCH_ADMIN) {
+        return AUTH_USER_TYPE.LAUNCH_ADMIN;
+      }
 
-    // Check if the user has any minimal admin organizations.
-    const minimalAdminOrgs = claims?.minimalAdminOrgs || {};
-    const hasMinimalAdminOrgs = Object.values(minimalAdminOrgs).some((org) => !_isEmpty(org));
+      if (claims?.role === UserRoles.ADMIN) {
+        return AUTH_USER_TYPE.ADMIN;
+      }
+    } else {
+      // Check if the user has any minimal admin organizations.
+      const minimalAdminOrgs = claims?.minimalAdminOrgs || {};
+      const hasMinimalAdminOrgs = Object.values(minimalAdminOrgs).some((org) => !_isEmpty(org));
 
-    if (hasMinimalAdminOrgs) {
-      return AUTH_USER_TYPE.ADMIN;
+      if (hasMinimalAdminOrgs) {
+        return AUTH_USER_TYPE.ADMIN;
+      }
```

Comment:
```suggestion
return hasMinimalAdminOrgs ? AUTH_USER_TYPE.ADMIN : undefined
```
Or return whatever else is appropriate, other than `undefined`.